Yoast/wordpress-seo
# #9342 Fix for 'duplicate content problem' link does not open in a new…

## Summary

This PR can be summarized in the following changelog entry:

* Added `target = "_blank"` to the anchor tag of the duplicate content link in the archive settings. 

## Test instructions

This PR can be tested by following these steps:

* Go to Seo->Search Appearance->Archives
* Click the question mark behind archives settings help
* Click the link duplicate content problem and see that it automatically open in a new tab.

## Quality assurance


Fixes #9342
